client/common/cmdline.c

```diff
@@ -5214,6 +5214,13 @@ static int freerdp_client_settings_parse_command_line_arguments_int(
 			if (rc != 0)
 				return rc;
 		}
+#ifdef WITH_VAAPI
```

Comment:
we already have `/gdi:hw` for hardware acceleration boolean, no need for this if you don´t require specific strings to be passed.